src/base.jl

```diff
@@ -1017,8 +1017,13 @@ end
 """
 Transform all instances of SingleTimeSeries to DeterministicSingleTimeSeries.
 """
-function transform_single_time_series!(sys::System)
-    IS.transform_single_time_series!(sys.data)
+function transform_single_time_series!(sys::System, horizon::Int, interval::Dates.Period)
```

Comment:
Thanks for fixing this oversight.  However, I think that we should follow the IS function signature and make the user pass in DeterministicTimeSeries.  This would be useful if we ever support a different kind of transformation.  However, if this is the only possible transformation we'll ever make, this is good.  Opinion @jd-lara ?

Reply:
This is probably the only transformation we will have to do. This is ok by me.

Reply:
@sourabhdalvi I think that Jose was saying that what you had originally was better...no type.